.gitignore

```diff
@@ -1,3 +1,4 @@
 target
 Cargo.lock
-tests/wasm/contract.wasm
\ No newline at end of file
+tests/wasm/*
+.vscode
```

Comment:
files that are in the `.gitignore` should also be removed if they were added with `git rm`
What I mean are:
- .vscode/
- tests/wasm/*
^ These weren't removed even after adding them to the list.